website/docs/en/blog/index.mdx

```diff
@@ -6,43 +6,49 @@ title: Overview
 
 Check here for the latest articles and release announcements about Rspack.
 
-## [Announcing Rspack v1.0 Alpha](/blog/announcing-1-0-alpha)
+## [Announcing Rspack 1.0](/blog/announcing-1-0)
+
+> August 28, 2024
+
+Today Rspack has finally reached a new milestone - 1.0. This means that Rspack is production ready, covers most of Webpack's APIs and features, and is now prepared to support more users.
```

Comment:
`Today Rspack has reached a new milestone - 1.0. This means that Rspack is production ready, covers most of webpack's APIs and features, and is now prepared to support more users.`

Summary of edits:
- webpack should be lower case according to their branding guidelines. 
- removing finally since it really didn't take that long. :)

Reply:
Nice catch, updated